corehq/apps/hqwebapp/static/hqwebapp/js/django.js

```diff
@@ -0,0 +1,10 @@
+/* globals define */
+/*
+    Workaround for integrating djangojs with requirejs.
+    djangojs.js creates a global django variable; the callback
+    option in requirejs config in style/base.html then sets that
+    global's gettext function to be a property of this module.
+*/
+define([], function() {
+    return {};
```

Comment:
also look at doing this with a shim instead?

Reply:
https://github.com/dimagi/commcare-hq/pull/17824/commits/0bc11a1230a3a020e01d5244a7adbfdebe1012cd

Did it without the shim, because the file path contains the language code (`/static/jsi18n/en/djangojs.js`), which seems unlikely to play nicely with the shim. Although, I could put the shim configuration in base.html so that python could insert the language code...I'm going to leave this alone for now, because it isn't obvious to me that one way or the other is better, but am open to more feedback.